src/fleet/azext_fleet/_help.py

```diff
@@ -23,6 +23,30 @@
           type: string
           short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the
                          managed cluster and resource group names.
+        - name: --enable-private-cluster
+          type: bool
+          short-summary: Prevents the Fleet's API server from be publicly accessed.
```

Comment:
from being*

Can you reuse the wording of the swagger documentation?
Swagger description is : "Whether to create the Fleet hub as a private cluster or not."

So,  this could be: "Create the Fleet hub as a private cluster."